### PR TITLE
Pull updated digital marketplace dependencies

### PR DESCRIPTION
**Pull request by bjgill**
Trello: https://trello.com/c/naKfCp2j/2108-however-when-i-click-the-link-im-getting-this

Some of them are quite outdated, which is causing problems with DOS 5. Update so admins can access DOS 5 things.

### CODE REVIEW

**Review by gidsg — APPROVED** [review]
👍 🆙

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-10.19.0
+10.23.0
```
Comment (gidsg) [inline .nvmrc:1]: I think a global version is expected by dmrunner so you may need to update this first

**Review by gidsg — CHANGES_REQUESTED** [review]

**Review by gidsg — APPROVED** [review]
Can you revert the node change for now as I think it'd be better to co-ordinate this with dmrunner, otherwise good to merge.

**Comment by bjgill** [comment]
Removed the node bump. Though dmrunner seemed perfectly happy with it for me...

Also remembered that I actually needed to run `npm install` and `make freeze-requirements`

**Comment by bjgill** [comment]
I've split the updates into their own commits to make it easier to debug. It turns out that it's the content loader update which is to blame.

I'm now out of time for today. If someone wants to pick this up on Monday, feel free. If you can work out what's wrong from the error, great. Otherwise, you'll probably want to bisect content loader versions until you work out which content loader version was the one that broke the tests.

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**